aiida/cmdline/commands/cmd_setup.py

```diff
@@ -53,47 +58,51 @@ def setup(
     profile.database_password = db_password
     profile.repository_uri = 'file://' + repository
 
+    CLI_LOGGER.debug('Loading config file')
```

Comment:
I could define a `echo_debug` instead, if that's preferred (perhaps cleaner?)

Reply:
Yes I would definitely be in favor of that

Reply:
yes, I will